src/input/index.js

```diff
@@ -38,6 +20,7 @@ class Input extends PureComponent {
     disabled: PropTypes.bool,
     readOnly: PropTypes.bool,
     activeDescendant: PropTypes.string,
+    searchInputLocation: PropTypes.oneOf(['dropdownToggle', 'dropdownContent']),
```

Comment:
How about we make this a bool? Something like `inlineSearchInput: true|false`, which is false by default. That way, users don't have to do anything if they desire existing behavior. And then structure the rest of the code based on that boolean?